### ISSUE TITLE
Flexslider Full Width (Drupal)

### ISSUE THREAD

**Issue by somasatori** [woothemes/FlexSlider#1329]
Hello! There doesn't seem to be a way to make FlexSlider 2 adjust to a full width in Drupal. I'm using the Corporate Clean theme with the nav blocks adjusted for full screen width. FlexSlider does this:
![flexslider example](https://cloud.githubusercontent.com/assets/10354734/7416467/2f2ca3e2-ef25-11e4-85b9-0040e43dbdd6.JPG)

I've adjusted a few things. For instance:

<code>.flexslider .slides img {
max-width: 1000%;
background: cover;
height: 100%;
}
</code>
&
<code>

.flexslider div li {
  background: cover;
  background-size: cover;
  padding-top: 6.57%; 
  width: 1000%;
  margin-left: -6.5%;
}

.flexslider div {
  width: 100%;
  background-size: cover;
  background-image: inherit;
}

.block-views-ssfp-view-block div {
  width: 100%;	

}
</code>

So basically I'm just blasting all of the elements with width: 100%, & background: cover; and attempting to use the images as background images. This was tried in flexslider.css in the library, as well as style.css in my subtheme. Drupal's not the most intuitive when it comes to markup within blocks, unfortunately. Any suggestions?

**Comment by somasatori** [comment]
Oh, the width: 1000%; listed in the .flexslider div li selector was in an attempt to knock out any type of additional mark up/CSS that might be working against my intent. I haven't modified the jquery script yet, since the place I work for is pretty adamant on it being fairly user intuitive and would rather not have a lot of back-end editing for creating new slideshows.

**Comment by somasatori** [comment]
I actually got this working. It was an issue with the theme region/block settings. No need to continue with this or follow up!

**Comment by alibama** [comment]
how'd you do it?
